api/context.go

```diff
@@ -50,6 +80,7 @@ func Transaction(ctx *gin.Context) {
 			rtx.DB = db
 			if len(ctx.Errors) > 0 {
 				err = ctx.Errors[0]
+				ctx.Errors = nil
```

Comment:
Clearing this fixes the error duplication that was seen on endpoints using the transaction handler.

Reply:
Ah.  good find.